spec/models/conference_spec.rb

```diff
@@ -1585,11 +1585,11 @@
   end
 
   describe 'after_create' do
-    let!(:conference) { Conference.new(title: 'ABC',short_title: 'XYZ',start_date: Date.today, end_date: Date.today + 10) }
-    let!(:free_ticket) { conference.tickets.first }
+    before { @conference = Conference.new }
```

Comment:
Yeah, and the thing is that I haven't included a free ticket in factory, so even after running all the sub models, the free ticket should come from the `after_create` callback, not from the factory.